src/current/cockroachcloud/ccloud-get-started.md

```diff
@@ -45,11 +45,18 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud auth login --org <organization label>
+ccloud auth login --org {organization label}
 ~~~
 
 The organization label is found on the **Settings** page of the CockroachDB Cloud Console.
 
+If your organization has a custom URL, use the `--vanity-name` flag to log in:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud auth login --vanity-name {custom organization URL}
```

Comment:
From what I can tell, the `--vanity-name` flag doesn't support the full organization URL (`https://cockroachlabs.cloud/login/acme`), but expects just the last part after `/login/` (`acme` in this example):

https://github.com/cockroachlabs/ccloud-private/blob/49beebd962ffec81e5ccac0cf0bbb5b2d0749f90/auth/auth_login.go#L81

`ccloud auth login --help` currently provides:

```
Flags:
  -h, --help                 help for login
      --no-redirect          disable redirection to a browser for login
      --org string           organization label
      --vanity-name string   vanity name, the last part of the custom URL for an org-specific login page
```

but I'm not sure what a clean way to say "the last part of the custom URL for an org-specific login page" would be 😅

Reply:
Does my suggested edit resolve this?

Reply:
Yep, that works for me.